app/static/admin/css/dashboard.rtl.css

```diff
@@ -0,0 +1,96 @@
+body {
```

Comment:
interesting, why the `rtl` css is added here?

Reply:
I'll remove that file